starcheck/state_checks.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from cxotime import CxoTime
+
+from astropy.table import Table, vstack
+import astropy.units as u
+from functools import lru_cache
+import numpy as np
+
+import kadi.commands as kadi_commands
+import kadi.commands.states as kadi_states
+from cxotime import CxoTime
+from chandra_maneuver import duration
+from Quaternion import Quat
+
+
+@lru_cache
+def make_man_table():
+    """
+    Calculate the time for range of manuever angles using Chandra.Maneuver
+    duration.
+    """
+    durations = []
+    q0 = Quat(equatorial=(0, 0, 0))
+
+    # Use a range of angles that sample the curve pretty well by eye
+    # The duration function is a little slow and not vectorized, so
+    # this is sparse.
+    angles = [0, 5, 10, 15, 20, 25, 35, 50, 100, 150, 180]
+    for angle in angles:
+        q1 = Quat(equatorial=(angle, 0, 0))
+        durations.append(duration(q0, q1))
+
+    return Table([durations, angles], names=["duration", "angle"])
+
+
+def calc_pcad_state_nman_sums(states):
```

Comment:
In general in this function you should replace `sum` with `duration`. "Sum" is ambiguous while "duration"  matches nomenclature in `cheta.utils.logical_states`.